common/src/main/java/com/wynntils/screens/itemsharing/ItemSharingScreen.java

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.screens.itemsharing;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.components.Handlers;
+import com.wynntils.core.components.Models;
+import com.wynntils.core.consumers.screens.WynntilsScreen;
+import com.wynntils.core.text.StyledText;
+import com.wynntils.models.items.FakeItemStack;
+import com.wynntils.models.items.WynnItem;
+import com.wynntils.models.items.encoding.type.EncodingSettings;
+import com.wynntils.models.items.items.game.CraftedConsumableItem;
+import com.wynntils.models.items.items.game.CraftedGearItem;
+import com.wynntils.models.items.items.game.GearItem;
+import com.wynntils.screens.base.widgets.WynntilsCheckbox;
+import com.wynntils.utils.EncodedByteBuffer;
+import com.wynntils.utils.colors.CommonColors;
+import com.wynntils.utils.mc.ComponentUtils;
+import com.wynntils.utils.mc.LoreUtils;
+import com.wynntils.utils.mc.McUtils;
+import com.wynntils.utils.render.FontRenderer;
+import com.wynntils.utils.render.RenderUtils;
+import com.wynntils.utils.render.Texture;
+import com.wynntils.utils.render.type.HorizontalAlignment;
+import com.wynntils.utils.render.type.TextShadow;
+import com.wynntils.utils.render.type.VerticalAlignment;
+import com.wynntils.utils.type.ErrorOr;
+import java.util.ArrayList;
+import java.util.List;
+import net.minecraft.ChatFormatting;
+import net.minecraft.client.gui.GuiGraphics;
+import net.minecraft.client.gui.components.AbstractWidget;
+import net.minecraft.client.gui.components.Button;
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.client.resources.language.I18n;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.item.ItemStack;
+
+public class ItemSharingScreen extends WynntilsScreen {
```

Comment:
I don't get it. This looks like a merge error. Was this not supposed to be added by the item sharing PR?

Reply:
I targetted this branch for the item sharing GUI since it uses the new encoding

Reply:
Aha, I see. I missed that part. Thanks.